plugins/module_utils/redfish_utils.py

```diff
@@ -732,14 +732,20 @@ def get_volume_inventory(self, systems_uri):
     def get_multi_volume_inventory(self):
         return self.aggregate_systems(self.get_volume_inventory)
 
-    def manage_indicator_led(self, command):
+    def manage_system_indicator_led(self, command):
+        return self.manage_indicator_led(command, self.systems_uri)
+
+    def manage_chassis_indicator_led(self, command):
+        return self.manage_indicator_led(command, self.chassis_uri)
+
+    def manage_indicator_led(self, command, resource_uri):
```

Comment:
```suggestion
    def manage_indicator_led(self, command, resource_uri=None):
```
and add something like
```
        if resource_uri is None:
            resource_uri = self.chassis_uri
```
Then this is a backwards compatbile change.